services/simcore-agent/.cookiecutterrc

```diff
@@ -0,0 +1,20 @@
+# This file exists so you can easily regenerate your project.
+#
+#  cookiecutter --overwrite-if-exists --config-file=.cookiecutterrc gh:pcrespov/cookiecutter-simcore-py-fastapi
```

Comment:
where is fastapi??

Reply:
not sure I see the point of keeping this file anyway.

Reply:
file was removed